config/ignition_fortress_ubuntu_jammy.yaml

```diff
@@ -4,15 +4,142 @@ suites: [jammy]
 component: main
 architectures: [amd64, armhf, arm64, source]
 filter_formula: "\
+(Package (= ignition-fortress) \
+), $Version (% 1.0.3-1*) |\
 (Package (= ignition-cmake2) |\
  Package (= libignition-cmake2-dev) \
-), $Version (% 2.10.0-99*) |\
+), $Version (% 2.12.1-1*) |\
+(Package (= ignition-common4) |\
+ Package (= libignition-common4) |\
+ Package (= libignition-common4-av) |\
+ Package (= libignition-common4-av-dev) |\
+ Package (= libignition-common4-core-dev) |\
+ Package (= libignition-common4-dev) |\
+ Package (= libignition-common4-events) |\
+ Package (= libignition-common4-events-dev) |\
+ Package (= libignition-common4-graphics) |\
+ Package (= libignition-common4-graphics-dev) |\
+ Package (= libignition-common4-profiler) |\
+ Package (= libignition-common4-profiler-dev) \
+), $Version (% 4.5.0+osrf-1*) |\
+(Package (= ignition-fuel-tools7) |\
+ Package (= libignition-fuel-tools7) |\
+ Package (= libignition-fuel-tools7-dev) \
+), $Version (% 7.0.0+osrf-1*) |\
+(Package (= ignition-gazebo6) |\
+ Package (= libignition-gazebo6) |\
+ Package (= libignition-gazebo6-dbg) |\
+ Package (= libignition-gazebo6-dev) |\
+ Package (= libignition-gazebo6-plugins) |\
+ Package (= python3-ignition-gazebo6) \
+), $Version (% 6.9.0-1*) |\
+(Package (= ignition-gui6) |\
+ Package (= libignition-gui6) |\
+ Package (= libignition-gui6-dev) \
+), $Version (% 6.4.0-1*) |\
+(Package (= ignition-launch5) |\
+ Package (= libignition-launch5) |\
+ Package (= libignition-launch5-dev) \
+), $Version (% 5.1.0-1*) |\
 (Package (= ignition-math6) |\
  Package (= libignition-math6) |\
  Package (= libignition-math6-dbg) |\
  Package (= libignition-math6-dev) |\
  Package (= libignition-math6-eigen3-dev) |\
  Package (= python3-ignition-math6) |\
  Package (= ruby-ignition-math6) \
-), $Version (% 6.10.0-99*) \
+), $Version (% 6.10.0-100*) |\
+(Package (= ignition-msgs8) |\
+ Package (= libignition-msgs8) |\
+ Package (= libignition-msgs8-dev) \
+), $Version (% 8.4.0-1*) |\
+(Package (= ignition-physics5) |\
+ Package (= libignition-physics5) |\
+ Package (= libignition-physics5-bullet) |\
+ Package (= libignition-physics5-bullet-dev) |\
+ Package (= libignition-physics5-core-dev) |\
+ Package (= libignition-physics5-dartsim) |\
+ Package (= libignition-physics5-dartsim-dev) |\
+ Package (= libignition-physics5-dev) |\
+ Package (= libignition-physics5-heightmap-dev) |\
+ Package (= libignition-physics5-mesh-dev) |\
+ Package (= libignition-physics5-sdf-dev) |\
+ Package (= libignition-physics5-tpe) |\
+ Package (= libignition-physics5-tpe-dev) |\
+ Package (= libignition-physics5-tpelib) |\
+ Package (= libignition-physics5-tpelib-dev) \
+), ($Version (% 5.1.0+osrf-2*) | $Version (% 5.1.0+osrf-3*)) |\
```

Comment:
This was manually changed after the scripted generator to account for a situation the script does not cover, where different platforms have different package revisions.